releases/v1.2.3.toml

```diff
@@ -0,0 +1,31 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.2.2"
+
+pre_release = false
+
+preface = """\
```

Comment:
I always like to see a friendly introduction here that mentions something about what is in the release as well as the severity of the changes.